Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Pages/Runtime.razor

```diff
@@ -0,0 +1,297 @@
+                        }
+                    </optgroup>
+                }
+            }
+        </select>
+        <label class="col-xs-3 control-label" for="function-payload">Function Input:</label>
+        <textarea class="form-control" style="margin: 5px" rows="20" @bind="FunctionInput" placeholder="JSON document as input to Lambda Function. Plain strings must be wrapped in quotes."></textarea>
+        <div class="col-sm form-group">
+            <button class="btn btn-primary btn-sm" @onclick="OnAddEventClick">Queue Event</button>
+        </div>
+    </div>
+    <div class="col-sm-6">
+        <h3>Active Event:</h3>
+        @if (RuntimeApiModel.ActiveEvent == null)
+        {
+            <h2>No active event</h2>
+        }
+        else
+        {
+                  <div>
+                      <div style="cursor: pointer" @onclick="() => OnRequeue(RuntimeApiModel.ActiveEvent.AwsRequestId)">
+                            @((MarkupString)REBOOT_ICON)
+                      </div>
+                      <p><b>Request ID:</b> @RuntimeApiModel.ActiveEvent.AwsRequestId</p>
+                      <p><b>Status:</b> <span style="@GetStatusStyle(RuntimeApiModel.ActiveEvent.EventStatus)">@RuntimeApiModel.ActiveEvent.EventStatus</span></p>
+                      <p><b>Last Updated:</b> @RuntimeApiModel.ActiveEvent.LastUpdated</p>
+                      <p><b>Event JSON:</b><span class="event-value"><span class="fake-link" @onclick="() => ShowEventJson(RuntimeApiModel.ActiveEvent)">@CreateSnippet(RuntimeApiModel.ActiveEvent.EventJson)</span></span></p>
+                      @if (RuntimeApiModel.ActiveEvent.EventStatus == IEventContainer.Status.Failure)
+                      {
+                          <p><b>Error Type:</b> @RuntimeApiModel.ActiveEvent.ErrorType</p>
+                          <p>
+                              <b>Error Response:</b>
+                              <pre class="form-control" style="@Constants.ResponseErrorStyleSizeConstraint" >@RuntimeApiModel.ActiveEvent.ErrorResponse</pre>
+                          </p>
+                      }
+                      else
+                      {
+                          <p>
+                              <b>Response:</b>
+                              <pre class="form-control" style="@Constants.ResponseSuccessStyleSizeConstraint">@Amazon.Lambda.TestTool.Utils.PrettyPrintJson(RuntimeApiModel.ActiveEvent.Response)</pre>
+                          </p>
+                      }
+                  </div>
+        }
+    </div>
+</div>
+
+
+<div class="row">
+    <div class="col-sm-6">
+        <h3>Queued Events: <button class="btn btn-secondary btn-sm" @onclick="OnClearQueued">Clear</button></h3>
+        <div class="col-xs-5 event-list">
+            @foreach (var evnt in @RuntimeApiModel.QueuedEvents)
+            {
+                <div class="event-list-item">
+                    <div class="row" style="padding: 2px">
```

Comment:
Should we be using pixel units instead of relative units for all of our padding? Bootstrap has built-in relative padding helpers and different ways of doing this. 2px on some screens is going to look better than others.